src/explore-education-statistics-admin/src/components/editable/EditablePageModeToggle.module.scss

```diff
@@ -1,11 +1,12 @@
 @import '~govuk-frontend/govuk/base';
-
+/* stylelint-disable max-nesting-depth */
 .toggle {
   background: $govuk-brand-colour;
   bottom: 0;
   left: 0;
-  padding: 0 6px 6px;
+  padding: 6px;
   position: fixed;
+  width: 390px;
```

Comment:
Does this need a specific width? If it's for the button, would flex work instead?

Reply:
I think it does, otherwise the button changes size when the content is hidden. I've tweaked it to be more responsive though